test/unit/test_Ites.cc

```diff
@@ -4,7 +4,8 @@
 #include <gtest/gtest.h>
 #include <Logic.h>
 #include <LRALogic.h>
-#include "../../src/itehandler/IteToSwitch.h"
+#include <IteToSwitch.h>
+#include <MainSolver.h>
```

Comment:
Including MainSolver does not seem to be necessary. I don't see any use of MainSolver in these tests.